src/unix/linux-core.c

```diff
@@ -193,9 +193,11 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
   int count;
   int nfds;
   int fd;
-  int op;
+  int op = 0;
   int i;
 
+  memset(&e, 0, sizeof(struct uv__epoll_event));
```

Comment:
What valgrind version and on what platform/architecture?  It's probably a false positive, valgrind doesn't always realize that system call arguments are sometimes in params, sometimes out and sometimes in/out.